bot.js

```diff
@@ -9,7 +9,7 @@ if (process.env.TEST) {
   console.log(`[${process.env.USER} => ${config.name}]`);
 } else {
   const IrcClient = require('./interfaces/irc');
-  var client = new IrcClient(config.server, config.name, config.channels);
+  client = new IrcClient(config.server, config.name, config.channels);
```

Comment:
Well this came as a redeclaration error while running lints. Declaring variables with `var` has function scope (not block scope). A better way would be declare the variable outside the blocks and assigning them in the `if` blocks. What do you say?